.github/workflows/end2end.yaml

```diff
@@ -524,9 +526,7 @@ jobs:
   end2end-https:
     needs: [build-kafka, build-test-image, check-dashboard-versions]
     runs-on:
-      - ubuntu
-      - focal
-      - xlarge
+      - ubuntu-22.04-8core
```

Comment:
do we actually need the larger workers?

Reply:
This move is about x2 in RAM and CPU, for a doubled price, see https://devdocs.scality.net/ci-engines/github-actions/runners/?h=ubuntu+22.04+8core#github-hosted-larger-pools-of-runners , as xlarge (see https://github.com/scality/devinfra/blob/main/manifests/runner-manager/overlays/prod/runner-manager.yaml#L91-L100 and https://cloud.google.com/compute/vm-instance-pricing#compute-optimized_machine_types for `e2-standard-4` ) is equivalent to ubuntu-22.04-4core
I'm totally fine doing it if this is the right move, but how do we jump to this conclusion exactly? (quickly looked at the ticket and the rest of the PR but I don't see anything obvious)

Reply:
For e2e-http and e2e-https we can probably use `ubuntu-22.04-4core` but we still need to jump to ubuntu 22.04 otherwise k8s bump and dependencies wont work..

Reply:
@KillianG then I would recommend to use a new runner but stay within the current specs; bumping a runner to a more powerful machine should be done if you experience network flakiness, memory issues, disk limits, ... stuff that should be visible in the logs